tests/torch/test_compression_training.py

```diff
@@ -458,81 +375,94 @@ def fixture_case_common_dirs(tmp_path_factory):
     }
 
 
-@pytest.mark.dependency(name="train")
-def test_compression_train(_params, tmp_path, case_common_dirs):
-    p = _params
-    args = p['args']
-    tc = p['test_config']
-
-    args['config'] = update_compression_config_with_legr_save_load_params(args['config'],
-                                                                          case_common_dirs["save_coeffs_path"])
-
-    args['mode'] = 'train'
-    args['log-dir'] = tmp_path
-    args['workers'] = 0  # Workaround for PyTorch MultiprocessingDataLoader issues
-    args['seed'] = 1
-    # Workaround for PyTorch 1.9.1 Multiprocessing issue related to determinism and asym quantization
-    # https://github.com/pytorch/pytorch/issues/61032
-    if 'mobilenet_v2_asym_int8.json' in args['config']:
-        args.pop('seed')
-
-    runner = Command(create_command_line(get_cli_dict_args(args), tc['sample_type'], tc['main_filename'],
-                                         tc['is_experimental']))
-    env_with_cuda_reproducibility = os.environ.copy()
-    env_with_cuda_reproducibility['CUBLAS_WORKSPACE_CONFIG'] = ':4096:8'
-    runner.kwargs.update(env=env_with_cuda_reproducibility)
-    runner.run(timeout=tc['timeout'])
-
-    checkpoint_path = os.path.join(args['checkpoint-save-dir'], tc['checkpoint_name'] + '_best.pth')
-    assert os.path.exists(checkpoint_path)
-    actual_acc = torch.load(checkpoint_path)['best_acc1']
-    ref_acc = tc['expected_accuracy']
-    better_accuracy_tolerance = 3
-    tolerance = tc['absolute_tolerance_train'] if actual_acc < ref_acc else better_accuracy_tolerance
-    assert actual_acc == approx(ref_acc, abs=tolerance)
-
-    # Check search results
-    checkpoint_path = os.path.join(args['checkpoint-save-dir'], tc['subnet_checkpoint_name'] + '.pth')
-    assert os.path.exists(checkpoint_path)
-    actual_acc = torch.load(checkpoint_path)['best_acc1']
-    ref_acc = tc['subnet_expected_accuracy']
-    assert actual_acc > ref_acc
```

Comment:
I wonder how you check search results in e2e test
where is `subnetwork_best` saved?